subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -703,6 +703,60 @@ The TestNG framework has a quite similar concept. In TestNG you can specify diff
             </sample>
 ++++
 
+[[migrate_to_junit5]]
+==== Migrate to JUnit 5
```

Comment:
I don't think we really need to mention "migration" in this context. Migration from JUnit 4 to 5 might only be one use case. For example a user may work on a new project that doesn't have any tests yet and decide for JUnit 5. Maybe a better title would be "Configuring JUnit 5 test execution". The anchor should be changed accordingly.

Reply:
Changed to "Using JUnit 5"